CMakeLists_files.cmake

```diff
@@ -50,6 +50,7 @@ list (APPEND PUBLIC_HEADER_FILES
         opm/flowdiagnostics/ConnectionValues.hpp
         opm/flowdiagnostics/ConnectivityGraph.hpp
         opm/flowdiagnostics/Toolbox.hpp
-        opm/flowdiagnostics/utility/AssembledConnections.hpp
-        opm/flowdiagnostics/utility/RandomVector.hpp
+        opm/utility/graph/tarjan.h
```

Comment:
While low-level, it is well documented, and it can be very useful. Also, it guards against the error of including it in a public header (by making it not an error...), something that can easily happen without too much notice (some class turning into a template is a likely source of such errors).